rustfmt.toml

```diff
@@ -0,0 +1,5 @@
+merge_imports = true
+
+ignore = [
+    "mobilecoin",   # The mobilecoin submodule is a dependency of this project.
```

Comment:
I think ignoring this is the right thing to do? Eventually, this project should probably depend on other mobilecoin crates instead of using a git submodule...